config/dspace_set_list.json

```diff
@@ -0,0 +1,1172 @@
+  "hdl_1721.1_7739": "Systems Design and Management - Master's degree",
+  "col_1721.1_7922": "Systems Design and Management - Ph.D. / Sc.D.",
+  "hdl_1721.1_7922": "Systems Design and Management - Ph.D. / Sc.D.",
+  "col_1721.1_7738": "Systems Design and Management - Ph.D. / Sc.D.",
+  "hdl_1721.1_7738": "Systems Design and Management - Ph.D. / Sc.D.",
+  "col_1721.1_127275": "USAID/Uganda Feed the Future Market System Monitoring Activity",
+  "hdl_1721.1_127275": "USAID/Uganda Feed the Future Market System Monitoring Activity",
+  "col_1721.1_16459": "Various Historical Departments",
+  "hdl_1721.1_16459": "Various Historical Departments",
+  "col_1721.1_114205": "Vegetable Cooling and Storage Evaluation in Mali",
+  "hdl_1721.1_114205": "Vegetable Cooling and Storage Evaluation in Mali",
+  "col_1721.1_18240": "Video and images",
+  "hdl_1721.1_18240": "Video and images",
+  "col_1721.1_7358": "Videos",
+  "hdl_1721.1_7358": "Videos",
+  "col_1721.1_3771": "Virtual Customer (VC)",
+  "hdl_1721.1_3771": "Virtual Customer (VC)",
+  "col_1721.1_114196": "Water Filter Evaluation in India",
+  "hdl_1721.1_114196": "Water Filter Evaluation in India",
+  "col_1721.1_114198": "Water Test Kit Evaluation in India",
+  "hdl_1721.1_114198": "Water Test Kit Evaluation in India",
+  "col_1721.1_114206": "Wheelchair Evaluation in Indonesia",
+  "hdl_1721.1_114206": "Wheelchair Evaluation in Indonesia",
+  "col_1721.1_91573": "White Papers",
+  "hdl_1721.1_91573": "White Papers",
+  "col_1721.1_40088": "Women's and Gender Studies (WGS) - Archived",
+  "hdl_1721.1_40088": "Women's and Gender Studies (WGS) - Archived",
+  "col_1721.1_34007": "Women's Studies (WMN) - Archived",
+  "hdl_1721.1_34007": "Women's Studies (WMN) - Archived",
+  "col_1721.1_33228": "Working papers",
+  "hdl_1721.1_33228": "Working papers",
+  "col_1721.1_18214": "Working Papers",
+  "hdl_1721.1_18214": "Working Papers",
+  "col_1721.1_34010": "Working Papers",
+  "hdl_1721.1_34010": "Working Papers",
+  "col_1721.1_80740": "Working Papers",
+  "hdl_1721.1_80740": "Working Papers",
+  "col_1721.1_88078": "Working Papers",
+  "hdl_1721.1_88078": "Working Papers",
+  "col_1721.1_18218": "Working Papers",
+  "hdl_1721.1_18218": "Working Papers",
+  "col_1721.1_26586": "Working Papers",
+  "hdl_1721.1_26586": "Working Papers",
+  "col_1721.1_18229": "Working Papers",
+  "hdl_1721.1_18229": "Working Papers",
+  "col_1721.1_26679": "Working Papers",
+  "hdl_1721.1_26679": "Working Papers",
+  "col_1721.1_108818": "Working Papers",
+  "hdl_1721.1_108818": "Working Papers",
+  "col_1721.1_97717": "Working Papers",
+  "hdl_1721.1_97717": "Working Papers",
+  "col_1721.1_18222": "Working Papers",
+  "hdl_1721.1_18222": "Working Papers",
+  "col_1721.1_92514": "Working Papers",
+  "hdl_1721.1_92514": "Working Papers",
+  "col_1721.1_26676": "Working Papers",
+  "hdl_1721.1_26676": "Working Papers",
+  "col_1721.1_18898": "Working Papers",
+  "hdl_1721.1_18898": "Working Papers",
```

Comment:
No idea. I'll add a ticket for stakeholder discussion and we can go from there.

Reply:
Hmm. Collections are just an array of strings in our index. What happens if you facet on a collection name that is duplicated? I assume it would just show anything that has at least one instance of, e.g., "Working Papers". So I'm not sure that's really going to cause a faceting problem? But I'm adding a ticket anyway as there are other stakeholder questions around DSpace collection names...